README.md

```diff
@@ -8,7 +8,80 @@ assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running the demo on Windows
 
+Prerequisites:
+- [Docker desktop](https://docs.docker.com/docker-for-windows/install/)
```

Comment:
On the download page I must pick between stable or edge channel. Would be nice to document which channel to use.
```suggestion
- [Docker desktop](https://docs.docker.com/docker-for-windows/install/) from stable channel
```